lib/compat/wordpress-5.9/register-global-styles-cpt.php

```diff
@@ -6,34 +6,44 @@
  * @package gutenberg
  */
 
-/**
- * Registers a Custom Post Type to store the user's origin config.
- *
- * This has been ported to src/wp-includes/post.php
+/*
+ * If wp_get_global_settings is defined, it means the plugin
+ * is running on WordPress 5.9, so don't need to register the CPT
+ * as it was already done by WordPress core.
  */
-function register_global_styles_custom_post_type() {
-	$args = array(
-		'label'        => __( 'Global Styles', 'gutenberg' ),
-		'description'  => 'CPT to store user design tokens',
-		'public'       => false,
-		'show_ui'      => false,
-		'show_in_rest' => false,
-		'rewrite'      => false,
-		'capabilities' => array(
-			'read'                   => 'edit_theme_options',
-			'create_posts'           => 'edit_theme_options',
-			'edit_posts'             => 'edit_theme_options',
-			'edit_published_posts'   => 'edit_theme_options',
-			'delete_published_posts' => 'edit_theme_options',
-			'edit_others_posts'      => 'edit_theme_options',
-			'delete_others_posts'    => 'edit_theme_options',
-		),
-		'map_meta_cap' => true,
-		'supports'     => array(
-			'title',
-			'editor',
-			'revisions',
-		),
-	);
-	register_post_type( 'wp_global_styles', $args );
+if ( ! function_exists( 'wp_get_global_settings' ) ) {
+
+	/**
+	 * Registers a Custom Post Type to store the user's origin config.
+	 *
+	 * This has been ported to src/wp-includes/post.php
+	 */
+	function register_global_styles_custom_post_type() {
+		$args = array(
+			'label'        => __( 'Global Styles', 'gutenberg' ),
+			'description'  => 'Global styles to include in themes.',
```

Comment:
In a second [commit](https://github.com/WordPress/gutenberg/pull/37282/commits/c801d70a20cc5f45e6dd46feb2d9e9d60a5f2679), I actually updated this text to be in line with [core's naming](https://github.com/WordPress/wordpress-develop/blob/9cc20468ef4803510f24caee93082514224f01f2/src/wp-includes/post.php#L451).